acmemanager.go

```diff
@@ -73,6 +73,12 @@ type ACMEManager struct {
 	// from this package
 	DNS01Solver acmez.Solver
 
+	// The solver for the http-01 challenge
+	HTTP01Solver acmez.Solver
+
+	// The solver for the tls-alpn-01 challenge
+	TLSALPN01Solver acmez.Solver
```

Comment:
Two things:

1. Can we move these above DNS01Solver? Sounds dumb, and it is dumb, but I usually see the order being: HTTP, TLSALPN, then DNS last; I just like to keep it consistent.

2. Can the godoc comments on these fields explain that these are _optional_ and that the default solvers work well for most people, including supporting distributed solving.